server.js

```diff
@@ -111,6 +111,21 @@ app.post('/editTask', (req, res) => {
   });
 });
 
+app.put('/updateStatus', (req, res) => {
```

Comment:
This task can be done using the /editTask method just above. I think you should use that. You can convert the post method to a put.

Reply:
Lets put this on hold for now. Make an issue regarding this so we remember